dexmaker/src/test/java/com/google/dexmaker/DexMakerTest.java

```diff
@@ -1814,11 +2003,11 @@ private Method getMethod() throws Exception {
         throw new IllegalStateException("no call() method");
     }
 
-    public static File getDataDirectory() throws Exception {
-        Class<?> environmentClass = Class.forName("android.os.Environment");
-        Method method = environmentClass.getMethod("getDataDirectory");
-        Object dataDirectory = method.invoke(null);
-        return (File) dataDirectory;
+    public static File getDataDirectory() {
```

Comment:
Yes, this change was to allow users to run unit tests without having to add various Android system framework jars to the classpath that vogar sets up (a somewhat non-trivial task, as I discovered). It should be backward compatible.

That said, I'll take this commit out of this pull request and put it in a separate one.